### ISSUE TITLE
Merits?

### ISSUE THREAD

**Issue by xgorinx** [Xeeynamo/KH3SaveEditor#12]
The save editor is amazing.  Is there any chance of being able to add a feature to modify PRO merits in the premium menu from the DLC?

**Comment by Kidel** [comment]
Seconding this. 
Just unlocked pro mode after beating Yozora but it seems that `All-rounder` and `Risk-taker` require a new game. Any way to speed things up would be greatly appreciated

**Comment by Veltro** [comment]
> 
> 
> Seconding this.
> Just unlocked pro mode after beating Yozora but it seems that `All-rounder` and `Risk-taker` require a new game. Any way to speed things up would be greatly appreciated

I know it's not related, but can you send me your savefile with Yozora unlocked @Kidel or I can send you mine so you can beat Luxord for me? Cause It's the only one I cannot win, and I want to unlock the trophy and Yozora so badly :/
Thanks.

**Comment by Kidel** [comment]
It's unrelated but I don't think my save would work for you, since I've already beaten the secret boss. And I'd prefer to avoid doing Luxord again (it was a pain). I suggest you add a new issue asking for a way to edit flags such as "Luxord beaten" so you can skip ahead and try the last secret boss.

Oh and please edit your comment to avoid major spoilers for others.

**Comment by Veltro** [comment]
Can you send me anyway in a Dropbox, GoogleDrive, etc? It's worth the try.
I just updated my comment and I already created the issue related to that. ;)
Thanks in advance.

**Comment by Kidel** [comment]
Please edit also my comment in your quote :) 
I've uploaded the binary in your issue. 

You'll see that doing `All-rounder` and `Risk-taker` however will be much worse.

**Comment by xgorinx** [comment]
Using the clear data from after beating Yozora to unlock pro codes and EZ codes, I was able to reload my clear file and fight each of the 13 in the dlc and yozora again. With only Kupo Coin turned off, I got 23,750 per battle (except yozora, which gave 28,500). A few I did with kupo coin turned on as well (ansem, luxord, larxene, dark riku, young xehanort), which each gave 25,000. Then I went back to the story mode version of the save and was able to fight armored xehanort and master xehanort and they each gave 10,000. So now I'm currently at 363,500 (B rank). Is there no way to re-do the Remind fights? Starting a new game erases the PRO merits progress, and I don't think I can no-kupo coin any more of these fights @.@

**Comment by xgorinx** [comment]
Well, I just confirmed that you can totally get this done without starting a new game.  If you just leave kupo coin off and turn on the other restrictions, then fight all the data battles, yozora, story mode xehanort, and dark inferno... you can reach the 375,000 points needed

**Comment by Kidel** [comment]
Do you still have the save at B rank?

**Comment by xgorinx** [comment]
> Do you still have the save at B rank?

Yup!  https://filebin.net/kxit6foogxubj433/Risk-Taker_DLC.zip?t=o7yommu8

Just go fight Dark Inferno (battlegate 14) on the main save (slot 1) and you should pop the trophy

**Comment by Veltro** [comment]
Thanks @xgorinx will help me too

**Comment by Kidel** [comment]
thanks @xgorinx 
US save it seems, I hope the bin is the same as EU version.

**Comment by Kidel** [comment]
@xgorinx can't open it in advanced mode using Save Wizard, would you mind exporting the bin of your slot 1?

**Comment by Kidel** [comment]
thanks @xgorinx

**Comment by xgorinx** [comment]
Glad it helped :)

**Comment by Xeeynamo** [comment]
@xgorinx I will see what I can do. Right now I did not completed the DLC, so I will need a bit of time to explore what is possible to modify and where those values are actually stored. I downloaded your save and I will use it as comparison 👍 thanks!

**Comment by Xeeynamo** [comment]
I honestly had no time to look into it with all the Final Fantasy VII Remake stuff :/
